### PR TITLE
Updated the slide control toolbar look

### LINKED ISSUE #5191: Update the look of the slide control toolbar

### PR DESCRIPTION
**Pull request by vitormateusalmeida**
Closes #5191

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/zoom-tool/component.jsx [hunk 1]
```diff
@@ -210,7 +210,7 @@ class ZoomTool extends Component {
               aria-describedby="zoomOutDesc"
               aria-label={intl.formatMessage(intlMessages.zoomOutLabel)}
               label={intl.formatMessage(intlMessages.zoomOutLabel)}
-              icon="minus"
+              icon="substract"
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/zoom-tool/component.jsx]: typo `subtract`?

Comment (vitormateusalmeida) [reply bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/zoom-tool/component.jsx]: https://github.com/bigbluebutton/bigbluebutton-font/blob/e51a8823dcbb04088b2ee1b933421e1d5b2efd7e/export/bbb-icons.js#L23

File: bigbluebutton-html5/imports/ui/components/presentation/component.jsx [hunk 2]
```diff
@@ -426,41 +433,30 @@ class PresentationArea extends Component {
...
+
+    const adjustedSizes = this.calculateSize();
+    const adjustedHeight = adjustedSizes.height;
+    const adjustedWidth = adjustedSizes.width;
+
+    const toolbarHeight = this.getToolbarHeight();
+
+    let toolbarWidth = 0;
+    if (this.refWhiteboardArea) {
+      const { clientWidth: AreaWidth } = this.refWhiteboardArea;
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/presentation/component.jsx]: Prefer `areaWidth` - changed capitalization

File: bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/component.jsx [hunk 3]
```diff
@@ -7,6 +7,8 @@ import Button from '/imports/ui/components/button/component';
 import { HUNDRED_PERCENT, MAX_PERCENT, STEP } from '/imports/utils/slideCalcUtils';
 import { styles } from './styles.scss';
 import ZoomTool from './zoom-tool/component';
+import FullscreenButton from '../../video-provider/fullscreen-button/component';
+import DownloadPresentationButton from '/imports/ui/components/presentation/download-presentation-button/component';
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/presentation/presentation-toolbar/component.jsx]: Unused?

**Pull request merged**